src/hcf-release/src/acceptance-tests-brain/test-resources/node-env/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "node-env",
   "version": "1.0.0",
   "engines": {
-    "node": "5.x"
   },
```

Comment:
We should either drop `engines` completely, or say `"engines": { "node": ">=5.0" }`, right?